### PR TITLE
79-employee-edit-bug : fixed employee edit bug and footer

### PR DESCRIPTION
**Pull request by MattKraatz**
## Status
**READY**

## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.

## Description
* Employee/Edit form should only disable Submit button if required fields are missing
* Footer text actually shows up now

## Resolves Issue Number
Resolves bugs #80 and #79 

## Deploy Notes

```sh
git pull --prune
git checkout 79-employee-edit-bug
dotnet run
```

## Steps to Test or Reproduce
See Description section above

## Impacted Files in Application
List general components of the application that this PR will affect:

* js/site.js
* Views/Shared/_Layout

### CODE REVIEW

File: src/workforce-management/wwwroot/js/site.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $(document).ready(function () {
-    if (location.pathname.includes("edit")) {
+    if (location.pathname.includes("Edit")) {
```
Comment (matthamil) [inline src/workforce-management/wwwroot/js/site.js:2]: good work

**Review by matthamil — APPROVED** [review]
:+1:

File: src/workforce-management/wwwroot/js/site.js (hunk 1 above)
Comment (daynewright) [reply src/workforce-management/wwwroot/js/site.js:2]: 🌮

**Comment by daynewright** [comment]
LGTM

**Comment by matthamil** [comment]
LGTM

**Comment by Anulfo** [comment]
LGTM

**Pull request merged**